src/main/resources/static/app/request/request-contents.html

```diff
@@ -217,7 +217,9 @@ <h4 class="h4">{{ 'Request details' | translate }}</h4>
             <div ng-if="request.statisticsRequest">{{'Numbers only, exploratory request'|translate}}<br /></div>
             <div ng-if="request.excerptsRequest">{{'Excerpts'|translate}}<br /></div>
             <div ng-if="request.paReportRequest" data-ng-bind-html="'PA reports' | translate"><br /></div>
-            <div ng-if="request.materialsRequest">{{'PA material'|translate}}<br /></div>
+            <div ng-if="request.blockMaterialsRequest">{{'PA material Block' | translate}}<br /></div>
+            <div ng-if="request.heSliceMaterialsRequest">{{'PA material HE slice(s)' | translate}}<br /></div>
+            <div ng-if="request.otherMaterialsRequest">{{request.otherMaterialsRequest}}<br /></div>
```

Comment:
Suggestion:
```
<div ng-if="request.otherMaterialsRequest">{{ 'PA material other' | translate }}: {{request.otherMaterialsRequest}}<br /></div>
```

Reply:
ah yes, I was wondering about this one, but I was not sure if with 'PA material other' it won't be taking too much space